android/app/src/main/java/com/gdgnitsurat/quickshare/utils/FirebaseUtil.kt

```diff
@@ -10,16 +14,31 @@ object FirebaseUtil {
 
     private var firebaseAuth: FirebaseAuth = FirebaseAuth.getInstance()
     private var firebaseUser: FirebaseUser? = firebaseAuth.currentUser
+    private val database: FirebaseDatabase = FirebaseDatabase.getInstance()
+    private var deviceid :String = ""
 
-    fun addCurrentUserToFirebaseDatabase() {
-        val database: FirebaseDatabase = FirebaseDatabase.getInstance()
-        val databaseRef: DatabaseReference = database.reference
-        val user = User(firebaseAuth.currentUser?.displayName, firebaseAuth.currentUser?.email)
-        databaseRef.child("users").child(firebaseAuth.currentUser?.uid).setValue(user)
+    fun addCurrentUserToFirebaseDatabase(deviceID:String) {
+        val databaseRef: DatabaseReference = database.getReference("users")
+        val user = User(firebaseAuth.currentUser?.displayName, firebaseAuth.currentUser?.email, deviceID)
+        databaseRef.child(firebaseAuth.currentUser?.uid).setValue(user)
+    }
+
+    fun addDevicesToFirebaseDatabase(deviceID:String) {
```

Comment:
Rename Devices -> Device :sweat_smile:

Reply:
Devices is cool because we have to add list of devices

Reply:
From one device it will be used once and ID will be same all the time

Reply:
i am talking about multiple login on multiple devices